content/20.pharmaceuticals.md

```diff
@@ -578,18 +604,24 @@ The effectiveness is summarized based on the current available evidence; large t
+Building on this, many within the field suggested that mechanism-informed phenotypic investigations may be the most useful approach to drug discovery [@doi:10.1038/nrd4366; @doi:10.1016/j.chembiol.2015.11.008; @doi:10.1126/scitranslmed.aab1201].
+As it stands, data-driven efforts to identify patterns in the results of computational screens allowed researchers to notice the shared property of cationic amphicility among many of the hits from computational screening analyses [@doi:10.1126/science.abi4708].
+While easier said than done, efforts to fill in the black box underlying computational HTS and recognize patterns among the identified compounds aid in moving data-oriented drug repurposing efforts in this direction.
+
+The unpredictable nature of success and failure in drug repurposing for COVID-19 thus highlights one of the tenets of phenotypic screening: there are a lot of "unknown unknowns", and a promising mechanism at the level of an MMOA will not necessarily propagate up to the pathway, cellular, or organismal level [@doi:10.1126/scitranslmed.aab1201].
+Despite the fact that apparently mechanistically relevant drugs may exist, identifying effective treatments for a new viral disease is extremely challenging.
+Targets of repurposed drugs are often non-specific, meaning that the MMOA can appear to be relevant to COVID-19 without a therapeutic or prophylactic effect being observed in clinical trials.
 The difference in the current status of remdesivir and favipiravir as treatments for COVID-19 (Table @tbl:summary-table) underscores how difficult to predict whether a specific compound will produce a desired effect, even when the mechanisms are similar.
 Furthermore, the fact that many candidate COVID-19 therapeutics were ultimately identified because of their shared propensity to induce phospholipidosis underscores how challenging it can be to identify a mechanism _in silico_ or _in vitro_ that will translate to a successful treatment.
-A very large number of papers using purely _in silico_ methods have been released [@url:https://blogs.sciencemag.org/pipeline/archives/2021/07/19/too-many-papers], many or most of which are unlikely to translate to _in vitro_ or _in vivo_ success.
-Computer-aided drug design and other computational analyses may provide value as a component of the broader drug discovery ecosystem when applied appropriately, but great care needs to be taken to ensure that they do not detract from the resources required to pursue more established approaches [@doi:10.1021/acs.jcim.0c00861].
+While significant progress has been made in the first 18 months of the pandemic, the therapeutic landscape is likely to continue to evolve as more results become available from clinical trials and as efforts to develop novel therapeutics for COVID-19 progress.
```

Comment:
"first 18 months" may be unclear because the start data is not precise.

Reply:
```suggestion
While significant progress has been made thus far in the pandemic, the therapeutic landscape is likely to continue to evolve as more results become available from clinical trials and as efforts to develop novel therapeutics for COVID-19 progress.
```
I want to say "year plus" but that probably isn't formal enough!